### PR TITLE
Convert status box components to TypeScript

### PR DESCRIPTION
**Pull request by spadgett**

### CODE REVIEW

File: frontend/public/components/utils/status-box.tsx [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    return <div className="loading-box loading-box__loaded">
+      {EmptyMsg ? <EmptyMsg /> : <EmptyBox label={label} />}
+    </div>;
+  }
+  return <div className="loading-box loading-box__loaded">{children}</div>;
+};
+Data.displayName = 'Data';
+
+export const StatusBox: React.FC<StatusBoxProps> = props => {
+  const {loadError, loaded, ...dataProps} = props;
```
Comment (jhadvig) [inline frontend/public/components/utils/status-box.tsx]: 👍

**Comment by spadgett** [comment]
@sg00dwin fyi, this probably conflicts with the changes you're working on

**Pull request merged**